docs/configuration.md

```diff
@@ -0,0 +1,350 @@
+| _unhealthyConditions_    | no                                    | `[{type: Ready, status: False, duration: 300s},{type: Ready, status: Unknown, duration: 300s}]` | List of UnhealthyCondition, which define node unhealthiness. See details below.                                                                                                                |
+
+### Selector
+
+The selector is selecting the nodes which should be observed. For its syntax have
+a look at the [LabelSelector docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements)
+
+A common selector for worker nodes looks like this:
+```yaml
+selector:
+  matchExpressions:
+    - key: node-role.kubernetes.io/control-plane
+      operator: DoesNotExist
+    - key: node-role.kubernetes.io/master
+      operator: DoesNotExist
+```
+The reason for excluding `master` and `control-plane` nodes instead of selecting
+`worker` nodes is, that this also prevents potentially unwanted remediation of
+control plane nodes in compact clusters, where nodes have both roles.
+
+For remediating control plane nodes use this:
+```yaml
+selector:
+  matchExpressions:
+    - key: node-role.kubernetes.io/control-plane
+      operator: Exists
+```
+> **Note**
+>
+> On older clusters you have to use `master` in above example. You can't use
+> both `master` and `control-plane` because the expressions are evaluated with
+> logical "AND".
+
+> **Warning**
+>
+> - Having a configuration which selects both worker and control plane nodes
+    > is strongly discouraged, because control plane nodes have special handling
+    > in NHC and potentially in remediators!
+> - Multiple configurations must not select an overlapping node set! This can lead to unwanted remediations.
+
+### RemediationTemplate
+
+The remediation template is an [ObjectReference](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/object-reference/)
+to a remediation template provided by a remediation provider. Mandatory fields
+are `apiVersion`, `kind`, `name` and `namespace`.
+
+> **Note**
+> 
+> This field is mutually exclusive with spec.EscalatingRemediations
+
+Note that some remediators work with the template being created in any namespace,
+others require it to be in their installation namespace.
+
+Also, some remediators install a remediation template by default, which can be
+used by NHC. This is e.g. the case for the SelfNodeRemediation remediator. The
+example CR above shows how to use its "resource deletion strategy" template.
+
+For other remediators you might need to create a template manually. Please check
+their documentation for details.
+
+For more details on the remediation template, and the remediation CRs created
+by NHC based on the template, see [below](#remediation-resources)
+
+### EscalatingRemediations
+
+EscalatingRemediations is a list of RemediationTemplates with an order and
```

Comment:
Nit: s/field/fields

Reply:
I don't think that's correct.
It's `one order field and one timeout field`, which is the same as `order and timeout field`, isn't it?
`Order and timeout fields` sounds like there can be  more than 2 fields to me...
This isn´t multiple subjects need plural verb...